testcases/OpTestEEH.py

```diff
@@ -36,140 +36,19 @@
 import sys
 import os
 
-from common.OpTestBMC import OpTestBMC
-from common.OpTestIPMI import OpTestIPMI
-from common.OpTestConstants import OpTestConstants as BMC_CONST
-from common.OpTestError import OpTestError
```

Comment:
Hrm... what I'll do is create an exception for the specific error cases that are thrown, that way we can detect failure and do recovery (e.g. re-ipl ?)